code/src/chunk.rs

```diff
@@ -13,12 +13,6 @@ pub struct Chunk {
 }
 
 impl Chunk {
-    /// Creates a new chunk from a validated `ChunkType` and some data.
-    /// The length and CRC will be computed automatically.
-    pub fn new(chunk_type: ChunkType, data: Vec<u8>) -> Self {
-        todo!()
-    }
-
```

Comment:
This is here so that the chapters feed into each other a little more cleanly. I want the reader to be able to use the `ChunkType` they created in the previous chapter directly.